Quaver.Shared/Screens/Select/SelectScreen.cs

```diff
@@ -351,6 +364,19 @@ private static void HandleKeyPressTab()
                 ConfigManager.LeaderboardSection.Value = LeaderboardType.Local;
         }
 
+        /// <summary>
+        ///     Handles when the user presses the del key
+        /// </summary>
+        private void HandleKeyPressDel()
+        {
+            var view = View as SelectScreenView;
+
+            if (!KeyboardManager.IsUniqueKeyPress(Keys.Delete) || DialogManager.Dialogs.Count > 0 || AvailableMapsets.Count == 0 || view.MapsetScrollContainer.SelectedMapsetIndex == -1)
```

Comment:
I think the `AvailableMapsets.Count == 0` is redundant now that there's a check for index = -1?